core/src/types/operator/metadata.rs

```diff
@@ -51,6 +51,10 @@ impl OperatorInfo {
     pub fn can_read(&self) -> bool {
         self.0.capability().read
     }
+    
+    pub fn can_read_with_override_content_disposition(&self) -> bool {
```

Comment:
We should not expose API like this. I prefer to have `fn capability(&self) -> Capability`